libminifi/include/core/ProcessSessionReadCallback.h

```diff
@@ -0,0 +1,33 @@
+#ifndef __PROCESS_SESSION_READ_CALLBACK_H__
```

Comment:
Since you broke this out into a different file ( thanks! ) it will need a license header.